probert/tests/test_multipath.py

```diff
@@ -0,0 +1,76 @@
+import mock
+import subprocess
+
+from probert import multipath
+from probert.tests.helpers import CiTestCase
+
+
+class TestMultipath(CiTestCase):
+
+    @mock.patch('probert.multipath.subprocess.run')
+    def test_multipath_show_paths(self, m_run):
+        mp_out = " ".join([self.random_string() for x in range(0, 8)])
+        cp = subprocess.CompletedProcess(args = ['foo'], returncode = 0,
+                                         stdout = mp_out.encode('utf-8'),
+                                         stderr = "")
+        m_run.return_value = cp
+        expected_result = [multipath.MPath(*mp_out.split())._asdict()]
+        result = multipath.multipath_show_paths()
+        self.assertEqual(expected_result, result)
+
+    @mock.patch('probert.multipath.subprocess.run')
+    def test_multipath_show_paths_skips_unparsable_output(self, m_run):
```

Comment:
The unparsable output here is just the empty string, IIRC. Should we add more lines with different garbage to make this test more robust, or are we intentionally only testing an empty line?

(If the latter, `test_multipath_show_paths_skips_blank_lines` might be a more apt name?)

Reply:
The input we parse here is from the multipathd show <paths|maps> with a formatter input.  In the case that multipathd doesn't have a value, sometimes it outputs "".  That makes me think that we should update the format to include the field names so we know which one was missing.  I'll do that in a separate PR and then update this unittest.